core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -227,14 +229,15 @@ object RequestChannel extends Logging {
 
   /** responseAsString should only be defined if request logging is enabled */
   class Response(val request: Request, val responseSend: Option[Send], val responseAction: ResponseAction,
-                 val responseAsString: Option[String]) {
+                 val responseAsString: Option[String], var throttledChannel: Option[ThrottledChannel]) {
```

Comment:
Can `throttledChannel` be `val` instead of `var`?

Reply:
Done